src/views/collection-view/helpers/format-sidebar-sections.ts

```diff
@@ -6,7 +6,7 @@ import { ListItemProps } from 'components/tutorials-sidebar/types'
 import { getCollectionSlug } from './get-slug'
 
 export interface CollectionCategorySidebarSection {
-	title?: CollectionCategoryOption
+	title?: string
```

Comment:
Opening this type up as the hcp sidebar doesn't use the same collection category options and I don't think it matters much if we're being super specific on the exact string for the title.